sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -342,39 +357,43 @@ def fetch_availability_zone_mappings() -> pd.DataFrame:
                         dest='az_mappings',
                         action='store_false')
     parser.add_argument(
-        '--check-regions-match-request',
+        '--strict-all-regions',
         action='store_true',
-        help=('Check whether the fetched data contains all regions '
-              'requested.'))
+        help=('Enforce the fetched data to contains all regions requested (all '
+              'regions or US regions).'))
```

Comment:
How about `--check-all-regions-enabled-for-account` (ok to be long since it's used in github actions only)?

Help: Check that this account has enabled "all" global regions hardcoded in this script. Useful to ensure our automatic fetcher fetches the expected data.